docs/deploy.rst

```diff
@@ -0,0 +1,244 @@
+   | Crate.io's Offering to connect Azure's EventHub to CrateDB. | consumer | eventhub-consumer | 1 - 3 (1 - 3 instances)   | s      |
+   | Crate.io's Offering to connect Azure's EventHub to CrateDB. | consumer | eventhub-consumer | 1 - 6 (1 - 6 instances)   | m      |
+   | Crate.io's Offering to connect Azure's EventHub to CrateDB. | consumer | eventhub-consumer | 1 - 18 (1 - 18 instances) | l      |
+   +-------------------------------------------------------------+----------+-------------------+---------------------------+--------+
+
+The output lists all available CrateDB products in your specified region. All
+products ship in four different Node sizes (node tiers).
+
+Pick the smallest tier which is the ``xs`` tier. In this case, you would be
+deploying the CrateDB cluster ``cratedb.az-gp1`` with the smallest tier.
+
+
+.. _launch-cluster:
+
+Launch a cluster
+================
+
+Now that you've set a region and chosen a product, you can deploy your first
+cluster.
+
+In CrateDB, clusters are deployed with the ``croud clusters`` command and some
+arguments to specify all necessary information for the cluster deployment.
+
+These are the required arguments:
+
++------------------------+-------------------------------------------------+
+| Argument               | Description                                     |
++------------------------+-------------------------------------------------+
+| ``--product-name``     | The product name to use for deployment.         |
++------------------------+-------------------------------------------------+
+| ``--tier``             | The product tier to use for deployment.         |
++------------------------+-------------------------------------------------+
+| ``--cluster-name``     | The CrateDB cluster name to use for deployment. |
++------------------------+-------------------------------------------------+
+| ``--project-id``       | The project ID to use for deployment.           |
++------------------------+-------------------------------------------------+
+| ``--version``          | The CrateDB version to use for deployment.      |
++------------------------+-------------------------------------------------+
+| ``--username``         | The CrateDB username to use for deployment.     |
++------------------------+-------------------------------------------------+
+| ``--password``         | The CrateDB password to use for deployment.     |
++------------------------+-------------------------------------------------+
+
+For this test deployment, the command should look like this:
+
+.. code-block:: console
+
+   sh$ croud clusters deploy \
+       --product-name cratedb.az-gp1 \
+       --tier xs \
+       --cluster-name my-first-crate-cluster \
+       --project-id 59c7961b-33a2-4478-9e98-2a6b3f54f60e \
+       --version 3.3.0 \
+       --username yourusername \
+       --password "yourpassword" \
+       --region bregenz.a1 \
+       --env prod
```

Comment:
you don't need to specify --env prod in the commands. the users should only have access to prod. other environments are only for cratedb cloud developers for testing.